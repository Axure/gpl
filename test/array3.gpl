void main()
{
	int[0] i_arr1;
	int[20] i_arr2;
	string[90] s_arr1;
	string[0] s_arr2;
	print(s_arr1.len());
	print(s_arr2.len());
	print(i_arr1.len());
	print(i_arr2.len());
}