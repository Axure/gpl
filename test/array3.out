90
0
0
20
